packages/machine-extractor/src/MachineExtractResult.ts

```diff
@@ -1469,6 +1513,95 @@ export class MachineExtractResult {
           this.machineCallResult.node.loc!.start.column,
     )!;
 
+    return {
+      range: oldRange,
+      newText: reprinted.slice(
+        machineNode.arguments[0].start!,
+        machineNode.arguments[0].end!,
+      ),
+      deleted: deleted.length ? deleted : undefined,
+    };
+  }
+
+  restore({ deleted }: { deleted: DeletedEntity[] }) {
+    // TODO deduplicate pre and post loop stuff
```

Comment:
Is this still relevant?

Reply:
it is in the sense that it could be done - but I don't intend to refactor this right now as in the ideal world I would just remove lots of this code altogether.